blockbase/css/front-end/style-base.css

```diff
@@ -0,0 +1,436 @@
+/* Text meant only for screen readers. */
+.screen-reader-text {
+	clip: rect(1px, 1px, 1px, 1px);
+	position: absolute !important;
+	height: 1px;
+	width: 1px;
+	overflow: hidden;
+	word-wrap: normal !important;
+	/* Many screen reader and browser combinations announce broken words as they would appear visually. */
+}
+
+.screen-reader-text:focus {
+	background-color: #f1f1f1;
+	box-shadow: 0 0 2px 2px rgba(0, 0, 0, 0.6);
+	clip: auto !important;
+	color: #21759b;
+	display: block;
+	font-size: var(--wp--typography--font-size);
+	font-size: 0.875rem;
+	font-weight: bold;
+	height: auto;
+	left: var(--wp--spacing--small);
+	line-height: normal;
+	padding: var(--wp--spacing--small);
+	text-decoration: none;
+	top: var(--wp--spacing--small);
+	width: auto;
+	z-index: 100000;
+	/* Above WP toolbar. */
+}
+
+/* Do not show the outline on the skip link target. */
+#primary[tabindex="-1"]:focus {
+	outline: 0;
+}
+
+/*--------------------------------------------------------------
+# Alignments
+--------------------------------------------------------------*/
+
+.alignleft,
+.alignright {
+	max-width: var(--wp--content-width--standard) !important;  /* Let's work to make this !important unnecessary */
+}
+
+.alignleft img,
+.alignright img,
+.alignleft figcaption,
+.alignright figcaption {
+	max-width: 50%;
+	width: 50%;
+}
+
+.alignleft figcaption {
+	clear: left;
+}
+
+.alignright figcaption {
+	clear: right;
+}
+
+.alignleft img,
+.alignleft figcaption {
+	float: left;
+	margin-right: var(--wp--spacing--medium);
+}
+
+.alignright img,
+.alignright figcaption {
+	float: right;
+	margin-left: var(--wp--spacing--medium);
+}
+
+.aligncenter {
+	clear: both;
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+/*--------------------------------------------------------------
+# Content
+--------------------------------------------------------------*/
+#page {
+	margin: 0 auto;
+	max-width: 100%;
+}
+
+/*--------------------------------------------------------------
+## Posts and pages
+--------------------------------------------------------------*/
+.page-content,
+.entry-content,
+.entry-summary {
+	margin: var(--wp--spacing--medium) 0 0;
+}
+
+.page-links {
+	clear: both;
+	margin: 0 0 var(--wp--spacing--medium);
+}
+
+.entry-header,
+.page-header,
+.entry-footer,
+.not-found .page-content,
+.search .entry-summary {
+	margin: var(--wp--spacing--medium) auto;
+	padding-left: var(--wp--spacing--small);
+	padding-right: var(--wp--spacing--small);
+	max-width: var(--wp--content-width--standard);
+}
+
+/*--------------------------------------------------------------
+# Media
+--------------------------------------------------------------*/
+
+/* Make sure embeds and iframes fit their containers. */
+embed,
+iframe,
+object {
+	max-width: 100%;
+}
+
+/*--------------------------------------------------------------
+# Custom Styles
+--------------------------------------------------------------*/
+
+@import url( "./css/custom.css" );
```

Comment:
I think this import is no longer needed given how you've restructured.

Reply:
Good catch! Should be all set now.